boards/arm/atsamd21_xpro/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+if(CONFIG_PINMUX_SAM0)
```

Comment:
Can you add copyright/license header info

Reply:
done